cmd/istioctl/main.go

```diff
@@ -299,15 +306,15 @@ func readInputs() ([]inputDoc, error) {
 			return nil, fmt.Errorf("could not encode Spec: %v", err)
 		}
 
-		schema, ok := model.IstioConfig[v.Type]
+		ischema, ok := model.IstioConfig[v.Type]
```

Comment:
The use of global variables (e.g. `schema`) makes this code harder to follow. It would be good to clean this up at some point, possibly in this PR or a follow-up PR.